### PR TITLE
Flake8 phase2

### PR DESCRIPTION
**Pull request by jochman**
fixes: https://app.circleci.com/pipelines/github/demisto/content/14986/workflows/31aa6a93-61b0-45de-b522-7406e78a6fb2/jobs/56786/parallel-runs/1?filterBy=FAILED

connected: https://github.com/demisto/content/pull/7338
- [x] Ready
AlienVaultOTX_v2
Cherwell
ThreatQ_v2

### CODE REVIEW

**Comment by jochman** [comment]
Circle fails on marketplace step - Ready to merge

File: Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.yml [hunk 1]
```diff
@@ -534,7 +534,8 @@ script:
     - contextPath: DBotScore.Vendor
       description: The vendor used to calculate the score.
       type: String
-  dockerimage: demisto/python3:3.7.4.1150
+  dockerimage45: demisto/python3:3.7.4.1150
```
Comment (Itay4) [inline Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.yml:537]: revert here too

**Comment by jochman** [comment]
Fail on Marketplace, can merge

**Comment by Itay4** [comment]
build fails on marketplace
merging

**Pull request merged**